src/couch/src/couch_att.erl

```diff
@@ -346,9 +346,14 @@ to_json(Att, OutputData, DataToFollow, ShowEncoding) ->
         {<<"content_type">>, Type},
         {<<"revpos">>, RevPos}
     ],
-    DigestProp = case base64:encode(Md5) of
-        <<>> -> [];
-        Digest -> [{<<"digest">>, <<"md5-", Digest/binary>>}]
+    DigestProp = case Md5 of
+        <<>> ->
```

Comment:
But you are changing logic here, right? Previously match for empty bin was on `base64` encode of `Md5` and here you are doing this on `Md5` itself. With this change an empty string will give you `<<"md5-">>, because `base64:encode(<<"">>) == <<>>`

Reply:
Nice catch!

Reply:
fixed in https://github.com/apache/couchdb/pull/3544/commits/ea6e208d0a30a8b9095ce38116afcc377f71429a